services/ext/api.go

```diff
@@ -442,22 +442,13 @@ func (api *PublicAPI) SetCommunityShard(request *requests.SetCommunityShard) (*p
 }
 
 // ExportCommunity exports the private key of the community with given ID
-func (api *PublicAPI) ExportCommunity(id types.HexBytes) (types.HexBytes, error) {
-	key, err := api.service.messenger.ExportCommunity(id)
-	if err != nil {
-		return nil, err
-	}
-	return crypto.FromECDSA(key), nil
+func (api *PublicAPI) ExportCommunity(id types.HexBytes) (*communities.CommunityImportInfo, error) {
+	return api.service.messenger.ExportCommunity(id)
 }
 
 // ImportCommunity imports a community with the given private key in hex
-func (api *PublicAPI) ImportCommunity(ctx context.Context, hexPrivateKey string) (*protocol.MessengerResponse, error) {
-	// Strip the 0x from the beginning
-	privateKey, err := crypto.HexToECDSA(hexPrivateKey[2:])
-	if err != nil {
-		return nil, err
-	}
-	return api.service.messenger.ImportCommunity(ctx, privateKey)
+func (api *PublicAPI) ImportCommunity(ctx context.Context, importData *communities.CommunityImportInfo) (*protocol.MessengerResponse, error) {
+	return api.service.messenger.ImportCommunity(ctx, importData)
```

Comment:
@richard-ramos, I'm getting a type signature failure because `api.service.messenger.ImportCommunity` still expects a `ecdsa.PrivateKey`.

Reply:
Forgot to add a change to the commit. Sorry!